wlog/wrappedlog/wrapped1log/context.go

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2018 Palantir Technologies. All rights reserved.
```

Comment:
nit: since this is a new file, copyright should be 2021 (I'm guessing this was just copy-pasted -- if you omit the header entirely, then the "verify" task will add the proper header with the current year as part of the "license" task)